projects/managed-hubs/incidents.md

```diff
@@ -0,0 +1,120 @@
+```
+
+## Communication channels
+
+### External communication
+
+- The {term}`Support Steward` team acts as the primary point of communication with external stakeholders like the {term}`Community Representative`s.
+- We may interact with external stakeholders via comments in Incident Response issues if it helps resolve the incident more quickly.
+
+### Internal communication
+
+- The Slack channel [{guilabel}`#support-freshdesk`](https://2i2c.slack.com/archives/C028WU9PFBN) contains real-time communication about support issues. Use this to signal-boost support requests related to {term}`Incident`s.
+- [Issues with the {guilabel}`incident` label](https://github.com/2i2c-org/infrastructure/issues?q=is%3Aopen+label%3A%22type%3A+Hub+Incident%22+sort%3Aupdated-desc) are where we track progress when [resolving incidents](support:incident-response).
+
+
+(support:incident-response)=
+## Incident response process
+
+Incidents are a special kind of support ticket, because they are related to degraded service that immediately impacts communities.
+We prioritize the resolution of incidents above all other kinds of work, and have a special process for tracking conversation and progress with them.
+
+Here is the process that we follow for incidents:
+
+1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Here is a template to get started:
+
+   ```
+   Hello { NAME }, we have investigated this request and have concluded that
+   it is related to an incident that is causing diminished service for your
+   community.
+   
+   We believe that this incident is related to { CONTEXT HERE } and will
+   investigate further on next actions. Information about our incident
+   response process can be found [in our team support documentation](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/support.html).
+
+   We will open an incident report issue in [our infrastructure repository](https://github.com/2i2c-org/infrastructure)
+   where you can track progress if you wish.
+ 
+   We'll prioritize resolving this incident over our other work, and
+   will communicate with you throughout our attempts to resolve it.
+   We might be in touch with requests for clarifications if needed.
+   ```
+2. **Open an incident issue**.
+   For each {term}`Incident` we create a dedicated issue to track its progress. [{bdg-primary}`open an incident issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+Hub+Incident%2Csupport&template=3_incident-report.md&title=%5BIncident%5D+%7B%7B+TITLE+%7D%7D) and notify our engineering team via Slack.
+3. **Try resolving the issue** and take notes while you gather information about it.
+4. **If after 30 minutes the issue is not solved or you know you cannot resolve it**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
```

Comment:
From the PM's perspective, the incident issue should be added to the sprint (cycle) and team backlogs, and the PM should do that.

Reply:
have added that in!